creator-node/tsconfig.json

```diff
@@ -1,31 +1,13 @@
+// This is used by `ts language service` and testing, so it needs to include the test folder
```

Comment:
what is `ts language service`?

Reply:
it's what VSCode uses to compile and find errors. so this config has types and files included to allow VSCode to know which default types exist (ex: if you do `describe` or `it` then it won't make an error)